Sources/Identity/CustomerInfoManager.swift

```diff
@@ -100,14 +98,6 @@ class CustomerInfoManager {
         }
     }
 
-    func sendCachedCustomerInfoIfAvailable(appUserID: String) {
```

Comment:
This wasn't being used by anything other than in tests.

Reply:
interesting! I guess it's a holder from this: https://github.com/RevenueCat/purchases-ios/pull/1828/

Reply:
Oh that's right 😇 this paves the way for that.